src/Abc.Zebus.Tests/Program.cs

```diff
@@ -12,9 +12,9 @@
 
 namespace Abc.Zebus.Tests
 {
-    public static class Program
+    public class Program
     {
-        public static void MainLol(string[] args)
+        public static void Main(string[] args)
```

Comment:
That will prevent tests from running on .NET Core, as the runner defines its own entry point.